components/02-molecules/menu/menu-toggle/menu-toggle.js

```diff
@@ -1,42 +1,135 @@
 Drupal.behaviors.menuToggle = {
   attach(context) {
-    // Selectors
+    // Selectors.
```

Comment:
Per our slack discussion you can likely trim down this file. 

- JS should calculate the the header height but assign that value to a CSS custom property.
- SCSS should use that custom property to determine the height of the mobile menu `height: calc(100vw - var(--heade-height);`
- Remove the media query JS. The frozen class should have no affect on desktop sizes which can be applied via a standard CSS media query

Reply:
I did all of this. 👍